nextflow.config

```
@@ -173,10 +173,12 @@ params {
     mapstats_preseq_defects_mode = false
 
     // Damage Calculation options
-    skip_damage_calculation      = false
-    damageprofiler_length        = 100
-    damageprofiler_threshold     = 15
-    damageprofiler_yaxis         = 0.3
+    skip_damage_calculation                     = false
+    damage_calculation_tool                     = 'damageprofiler'
+    damage_calculation_yaxis                    = 0.3
+    damage_calculation_damageprofiler_length    = 100
+    damage_calculation_damageprofiler_threshold = 15
+    damage_calculation_mapdamage_downsample     = 0
```

```suggestion
    damagecalculation_tool                     = 'damageprofiler'
    damagecalculation_yaxis                    = 0.3
    damagecalculation_damageprofiler_length    = 100
    damagecalculation_damageprofiler_threshold = 15
    damagecalculation_mapdamage_downsample     = 0
```

We should make sure we are consistent (including myself)

<sectionsection>_<tool>_<parameter>

---

Would need to be updated everywhere